### PR TITLE
Add wager slider

### PR DESCRIPTION
**Pull request by jakeane**
# Title

To change how wagers are created, we will use a slider to set the amount a user wants to wager, ranging from $1 to $10. Given that a lot of the wager layout is going to change, not much was done with relative layout. That will be addressed in #168.

## Type of Change

- [x] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Documentation update

## Checklist

- [ ] Unit/integration tests
- [ ] Documentation

## Tickets

- [#147](https://app.zenhub.com/workspaces/betmate-60612106ee6e6e000ee3aad2/issues/dali-lab/betmate-frontend/147)

## Screenshots

Screen Shot 2021-08-10 at 10 53 52 PM
Screen Shot 2021-08-10 at 10 54 21 PM

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -3,12 +3,12 @@
 // eslint-disable-next-line no-nested-ternary
 export const ROOT_URL = process.env.TARGET_ENV === 'prod'
   ? 'https://betmate-backend-prod.herokuapp.com'
-  : process.env.TARGET_ENV === 'dev'
+  : process.env.TARGET_ENV === 'dev' || true
     ? 'https://betmate-backend-dev.herokuapp.com'
```
Comment (jakeane) [inline src/utils/index.ts:7]: The next few changes will just be UI related, so I will be keeping this through the upcoming PRs

File: src/utils/index.ts [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
     : 'http://localhost:9090';
 
 // Auth token name for storage and transmission to backend
 export const authTokenName = 'authToken';
 
 // Number of ms before an axios request times out
-export const requestTimeout = 1000;
+export const requestTimeout = 3000;
```
Comment (jakeane) [inline src/utils/index.ts:14]: This hopefully resolves some of the timeout issues

Comment (tvergho) [reply src/utils/index.ts:14]: an axios retry package might also be helpful to automatically retry requests a set number of times when they timeout

**Review by tvergho — APPROVED** [review]

File: src/utils/index.ts (hunk 2 above)
Comment (jakeane) [reply src/utils/index.ts:14]: Yeah that may be worth looking into

**Pull request merged**